### PR TITLE
[ENH] :sparkles: new page applicationsSearch

### PR DESCRIPTION
**Pull request by Donatien26**
Signed-off-by: Donatien ENEMAN <donatien.eneman@insee.fr>

### CODE REVIEW

File: src/components/pages/searchApplications/applications_favorite_list.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+					spacing={3}
+				>
+					{loading ? (
+						<LinearProgress />
+					) : applications.length > 0 ? (
+						applications
+							.slice(
+								(page - 1) * rowsPerPage,
+								(page - 1) * rowsPerPage +
+									rowsPerPage,
```
Comment (clement-dufaure) [inline src/components/pages/searchApplications/applications_favorite_list.tsx:61]: page * rowsperpage ?

File: src/components/pages/searchApplications/applications_list.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+										)}
+									</TableCell>
+								</TableRow>
+							) : (
+								applications
+									.slice(
+										(page - 1) *
+											rowsPerPage,
+										(page - 1) *
+											rowsPerPage +
```
Comment (clement-dufaure) [inline src/components/pages/searchApplications/applications_list.tsx:88]: page * rowsPerPage

File: src/components/pages/searchApplications/applications_favorite_list.tsx (hunk 1 above)
Comment (Donatien26) [reply src/components/pages/searchApplications/applications_favorite_list.tsx:61]: Ahah yes 😅

File: src/components/pages/searchApplications/applications_viewer.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+										/>
+									),
+								)}
+						</Typography>
+					);
+				},
+			},
+		},
+		{
+			name: 'groups',
```
Comment (clement-dufaure) [inline src/components/pages/searchApplications/applications_viewer.tsx:73]: groups not available at this step, please remove column to avoid confusion

**Review by clement-dufaure — APPROVED** [review]

**Pull request merged**